### DISCUSSION TITLE
UX of updating lazer is bad

### DISCUSSION THREAD

**Discussion by Glinte** [ppy/osu#19058] [Ideas] [answered]
On the side bar, updating already requires 2 button clicks (download the update, and actually updating). But upon clicking update, it doesn't just restarts osu! and instead pops up the exit window. The text is also identical to exitting via alt+f4 (are you sure you want to exit osu!?), and have no indication that this is a popup relating to the update. In first time that I have updated, I thought I somehow accidentally clicked something that causes osu to close. Another problem is that before switching to the new squirrel, the exit window would auto choose let me out after a few seconds but stopped doing this after switching.

I think it would be best if lazer just closes itself after I click update on the side bar.

**Comment by peppy** [comment, marked as answer]
The confirmation on exit will eventually only show when there's something to be confirmed. Also, there's multiple other threads already discussing thus such as https://github.com/ppy/osu/discussions/13059.

**Comment by Glinte** [comment]
My apologies, I couldn't find a good keyword to search threads about this. I have searched "update" and read 5 pages but there are too many unrelated discussions. Sorry for duplicating.
